Packs/Legacy/Scripts/script-GrrGetFiles_CHANGELOG.md

```diff
@@ -1,2 +0,0 @@
-## [Unreleased]
```

Comment:
why delete and not move?
the same question for the other deletes here

Reply:
dupliacted (hope they're the last ones)
![image](https://user-images.githubusercontent.com/38749041/81784610-ca527c80-9505-11ea-94b6-2139ba62b8fc.png)
also `Packs/Legacy/Scripts/script-NetwitnessQuery_CHNAGELOG.md` is redundant (notice the typo in `CHNAGELOG`

Reply:
@dantavori this list is longer than the list of deleted files in the PR